lib/openstax_rescue_from.rb

```diff
@@ -1,29 +1,28 @@
-ActiveSupport::Inflector.inflections do |inflect|
-  inflect.acronym 'OpenStax'
-end
-
-require "openstax/rescue_from/configuration"
-require "openstax/rescue_from/version"
+require 'openstax/rescue_from/engine'
+require 'openstax/rescue_from/wrapped_exception'
 
 module OpenStax
   module RescueFrom
+    def self.included(base)
```

Comment:
I would suggest keeping modules to be included separate from the main gem module.